Explain how to enable profiling for Go programs built with Orchestrion (#24912)
DataDog/documentation: content/en/profiler/enabling/go.md

**Note**: By default, only the CPU and Heap profiles are enabled. Use [profiler.WithProfileTypes][11] to enable additional [profile types][12].

> If you have automatically instrumented your Go application with [Orchestrion][20],
> the continuous profiler code will already be added to your application.
> Enable the profiler at run time by setting the environment variable `DD_PROFILING_ENABLED=true`.

Reviewer: ```suggestion
If you automatically instrumented your Go application with [Orchestrion][20], it already added the continuous profiler code to your application. To enable the profiler at run time, set the environment variable `DD_PROFILING_ENABLED=true`.
```
